.eslintrc.js

```diff
@@ -2,7 +2,6 @@ module.exports = {
   extends: "airbnb-base",
   plugins: ["import", "prettier"],
```

Comment:
since we removed `prettier` from `eslint` rules, ¿should we remove/keep it here?